pom.xml

```diff
@@ -1978,6 +1978,45 @@
               </execution>
             </executions>
           </plugin>
+
+          <plugin>
+            <groupId>de.thetaphi</groupId>
+            <artifactId>forbiddenapis</artifactId>
```

Comment:
Can we reuse existing instance of this plugin https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L1540

Reply:
my bad, I didn't notice. fixed now.